server/unittools.cpp

```diff
@@ -2559,18 +2557,9 @@ void kill_unit(struct unit *pkiller, struct unit *punit, bool vet)
     /* Inform the destroyer again if more than one unit was killed */
     if (unitcount > 1) {
       notify_player(pvictor, unit_tile(pkiller), E_UNIT_WIN_ATT, ftc_server,
-                    /* TRANS: "... Cannon ... the Polish Destroyer ...." */
-                    PL_("Your attacking %s succeeded against the %s %s "
-                        "(and %d other unit)!",
-                        "Your attacking %s succeeded against the %s %s "
-                        "(and %d other units)!",
-                        unitcount - 1),
-                    pkiller_link, nation_adjective_for_player(pvictim),
-                    punit_link, unitcount - 1);
-    }
-
-    if (vet) {
-      notify_unit_experience(pkiller);
+                    /* TRANS: "Another unit was eliminated by..." */
+                    "Another %s eliminated by our attacking %s!",
+                    unit_n_stack_kills(unitcount), punit_link);
```

Comment:
This is difficult to translate, better a `PL_()`:
* `unitcount==1`: Un autre unité a été éliminée dans l'attaque de notre Croiseur!
* `unitcount>1`: **2 autres** unité**s** **ont** été éliminée**s** dans l'attaque de notre Croiseur!

It is generally advisable not to break sentences like this because this tends to assume English grammar.

Reply:
I used PL_ elsewhere, but I couldn't quite figure out how to make it work here. Any ideas?

Reply:
Hmm I suppose the issue that the `%d` isn't always there. There's [a trick](https://stackoverflow.com/a/16765923) for that (first 3 lines, the rest is untranslatable), which is to move the optional parameter to the end:
```cpp
notify_player(...
    PL_("Another unit eliminated by our attacking %s!",
        "Another %2$d units eliminated by our attacking %1$s!",
         unitcount - 1),
    punit_link, unitcount - 1);
```